commands_function.py

```diff
@@ -0,0 +1,49 @@
+import os
+from abc import ABCMeta,abstractclassmethod, abstractstaticmethod
+import json
+
+class Icommand(metaclass=ABCMeta):
+    @abstractstaticmethod
+    def excute():
+        """the interface Icommand"""
+
+class commandfactory:
+    @staticmethod
+    def build_command(args):
+        command_type=args.command
+        if command_type == "create":
+            return create()
+        elif command_type == "delete":
+            return delete()
+        elif command_type == "get":
+            return get()
+        elif command_type == "set":
+            return set()
+
+class create(Icommand):
```

Comment:
I recommend every command class in a file. 
u will have create_command.py
get_command.py. 

also class name should be CreateCommand. 


class name should start with capital